### PR TITLE
Remove usage of augur full dependency list

### PR DESCRIPTION
**Pull request by victorlin**
### Description of proposed changes

It has been removed in https://github.com/nextstrain/augur/pull/1035.

### Testing

- [x] Checks pass

### CODE REVIEW

**Review by tsibley — APPROVED** [review]
Good to go once https://github.com/nextstrain/augur/pull/1035 is merged.

**Comment by victorlin** [comment]
Since the Dockerfile [downloads the latest Augur release](https://github.com/nextstrain/docker-base/blob/3d0a439b36615c182e672e0fe9250405317473dd/Dockerfile#L87), I'll wait for a new Augur release before merging this.

**Comment by tsibley** [comment]
Ah, indeed. I'd forgotten that Augur was latest release not `master`!

**Pull request merged**
